### PR TITLE
logging: Move from logzero to logging

### PR DESCRIPTION
**Pull request by odscjames**
https://github.com/open-contracting/kingfisher-views/issues/93
https://github.com/open-contracting/kingfisher-views/issues/79

### CODE REVIEW

**Comment by odscjames** [comment]
[Store outputs of add-view, refresh-views and field-counts; migrate away from logzero](https://trello.com/c/T7uXC7kD/208-store-outputs-of-add-view-refresh-views-and-field-counts-migrate-away-from-logzero)

File: docs/logging.rst [hunk 1]
```diff
@@ -0,0 +1,31 @@
+Logging
+=======
+
+If logging is configured using a :ref:`default configuration<config-logging>`, then log messages are written to ``info.log`` (and possibily ``debug.log``).
```
Comment (jpmckinney) [inline docs/logging.rst:4]: possibily -> possibly (I now fixed in Process)

File: docs/logging.rst [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+ocdskingfisher.views.config
+  An ``WARN``-level message if there was a problem loading the config.
+ocdskingfisher.views.field-counts
+  ``INFO``-level for the process of running field counts.
+ocdskingfisher.views.refresh-views
+  ``INFO``-level for the process of running refresh views.
+ocdskingfisher.views.cli
+  An ``INFO``-level message whenever a CLI command is run, by a user or by `cron <https://en.wikipedia.org/wiki/Cron>`__.
+ocdskingfisher.views.cli.delete-view
+  ``INFO``-level for the process of deleting a view.
```
Comment (jpmckinney) [inline docs/logging.rst:31]: Prefer consistent style ("An INFO-level message for" not just "INFO-level for") to make vertical scanning easier.

File: ocdskingfisherviews/cli/commands/delete_view.py [hunk 3]
```diff
@@ -1,6 +1,6 @@
-
+import logging
 import sqlalchemy as sa
-from logzero import logger
+
```
Comment (jpmckinney) [inline ocdskingfisherviews/cli/commands/delete_view.py:3]: Extra newline.

File: ocdskingfisherviews/cli/__init__.py [hunk 4]
```diff
@@ -1,10 +1,22 @@
...
+import json
+import logging
+import logging.config
+import os
 
 import ocdskingfisherviews.cli.util
 
 
 def run_command(input_args=None):
+
```
Comment (jpmckinney) [inline ocdskingfisherviews/cli/__init__.py:12]: Prefer no newline after `def` line.

**Comment by jpmckinney** [comment]
[Views: Document where we can determine its status](https://trello.com/c/BohEHJFi/51-views-document-where-we-can-determine-its-status)

**Pull request merged**